analysis/src/Hover.ml

```diff
@@ -2,6 +2,36 @@ open SharedTypes
 
 let codeBlock code = Printf.sprintf "```rescript\n%s\n```" code
 
+let encodeURIComponent text =
```

Comment:
This needs to be replaced by something vastly more complete/robust. Hopefully there's something for OCaml we can find and vendor or similar.